app/controllers/bulk_ingest_controller.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 class BulkIngestController < ApplicationController
```

Comment:
A lot of things are called `uploads`. There's new uploads and pending uploads and files and containers, I wonder if you could explain the differences?